Assets/_Game/Scripts/Enemy/EnemyHealth.cs

```diff
@@ -20,31 +25,30 @@ public class EnemyHealth : MonoBehaviour, IPooledObject {
     if (enemy == null) {
       enemy = GetComponent<Enemy>();
     }
-
   }
 
   private void Start() {
     currentHealth = enemyLoader.maxHealth;
   }
 
   public void MakeDead() {
-    ObjectPool.Instance.ReturnToPool(Constants.Tag_Enemy,gameObject);
+    ObjectPool.Instance.ReturnToPool(Constants.Tag_Enemy, gameObject);
```

Comment:
Move enemyList to ObjectPool and clear when returning to pool or reload the scene